{{ cookiecutter.project_name }}/requirements.txt

```diff
@@ -2,3 +2,12 @@ pytest
 pre-commit
 cookiecutter
 mkdocs
+
+{%- if cookiecutter.framework_if_using_python=="pytorch" -%}
+pytorch
```

Comment:
Can we include version of the framework as an option when launching it? Or is that unnecessary?